docs/static/ds6/skin.css

```diff
@@ -2881,42 +2881,66 @@ span.textbox__icon:first-child,
 .tooltip--tooltip,
 .tooltip--infotip,
 .tooltip--tourtip {
+  box-shadow: 0 2px 4px 1px #c7c7c7;
   font-size: 0.875rem;
   padding: 16px;
   position: absolute;
   z-index: 1;
 }
-.tooltip--tooltip,
-.tooltip--tourtip {
-  background-color: #006efc;
-  color: #fff;
-}
-.tooltip--infotip {
-  background-color: #fff;
-  box-shadow: 0 0 3px 1px #767676;
-  color: #111820;
-}
+.tooltip--tooltip .arrow-top-left,
 .tooltip--infotip .arrow-top-left,
+.tooltip--tourtip .arrow-top-left,
+.tooltip--tooltip .arrow-top-center,
 .tooltip--infotip .arrow-top-center,
-.tooltip--infotip .arrow-top-right {
-  box-shadow: -2px -2px 2px #767676;
+.tooltip--tourtip .arrow-top-center,
+.tooltip--tooltip .arrow-top-right,
+.tooltip--infotip .arrow-top-right,
+.tooltip--tourtip .arrow-top-right {
+  box-shadow: -2px -2px 2px #c7c7c7;
 }
+.tooltip--tooltip .arrow-bottom-left,
 .tooltip--infotip .arrow-bottom-left,
+.tooltip--tourtip .arrow-bottom-left,
+.tooltip--tooltip .arrow-bottom-center,
 .tooltip--infotip .arrow-bottom-center,
-.tooltip--infotip .arrow-bottom-right {
-  box-shadow: 2px 2px 2px #767676;
+.tooltip--tourtip .arrow-bottom-center,
+.tooltip--tooltip .arrow-bottom-right,
+.tooltip--infotip .arrow-bottom-right,
+.tooltip--tourtip .arrow-bottom-right {
+  box-shadow: 2px 2px 2px #c7c7c7;
 }
+.tooltip--tooltip .arrow-left-top,
 .tooltip--infotip .arrow-left-top,
+.tooltip--tourtip .arrow-left-top,
+.tooltip--tooltip .arrow-left-center,
 .tooltip--infotip .arrow-left-center,
-.tooltip--infotip .arrow-left-bottom {
-  box-shadow: -2px 2px 2px #767676;
+.tooltip--tourtip .arrow-left-center,
+.tooltip--tooltip .arrow-left-bottom,
+.tooltip--infotip .arrow-left-bottom,
+.tooltip--tourtip .arrow-left-bottom {
```

Comment:
Hmmm. Seems to have bloated. Can you have a look as to your selector nesting in the LESS?

This should really be simply:
```
.tooltip .arrow-left-top,
.tooltip .arrow-left-center,
.tooltip .arrow-left-bottom { ... }
````
Thoughts?